pkg/console/operator/sync_v400.go

```diff
@@ -358,6 +364,41 @@ func (co *consoleOperator) SyncConfigMap(
 	return cm, cmChanged, "ConsoleConfigBuilder", cmErr
 }
 
+func (co *consoleOperator) SyncClusterCAConfigMaps(ctx context.Context, operatorConfig *operatorv1.Console, recorder events.Recorder) ([]*corev1.ConfigMap, bool, string, error) {
+	managedClusters, err := co.managedClusterClient.ManagedClusters().List(ctx, metav1.ListOptions{LabelSelector: fmt.Sprintf("local-cluster!=true")})
+	if err != nil || len(managedClusters.Items) == 0 {
+		return nil, false, "", nil
+	}
+
+	configMaps := []*corev1.ConfigMap{}
```

Comment:
Let's give the variable more descriptive name here. Maybe `clusteCAConfigMaps` ?